Minor rewording on upgrade (#1056)
StackStorm/st2docs: docs/source/install/upgrades.rst

.. _migration-scripts-to-run:

> Version-specific Changes / Migration Scripts

Reviewer: ```suggestion
Version-Specific Changes / Migration Scripts
```

v3.4
''''

> *  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3. All packs installed prior to upgrade on Ubuntu 16 and RHEL/CentOS 7 will need to have their virtualenvironment re-created after upgrade, using the ``packs.setup_virtualenv`` action.

Reviewer 2: Maybe we can also document / link to the ``sudo st2ctl reload --register-setup-virtualenvs`` option (that's the one I assume in "gitops" / infra as code manner).

v3.4
''''

*  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3. All packs installed prior to upgrade on Ubuntu 16 and RHEL/CentOS 7 will need to have their virtualenvironment re-created after upgrade, using the ``packs.setup_virtualenv`` action:

.. sourcecode:: bash

>     st2 run packs.setup_virtualenv packs=$(st2 pack list -a ref -j | jq -r '. | map(.ref) | join(",")')

Reviewer 3: Per discussion in Slack based on @Kami suggestion,
`st2ctl register --register-setup-virtualenvs` would do the same and is simpler & cleaner.

v3.4
''''

> *  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3. 

Reviewer: ```suggestion
*  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
```

    sudo yum install python3-devel --enablerepo <optional-server-rpm repo>

> * *Ubuntu 16.04 Xenial only.* Python 3.6 is not available in the base Ubuntu Xenial distribution. Python 3.6 must be available before you upgrade |st2| packages, and you can add unofficial 3rd party `Python PPA repository <https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa>`_:

Reviewer: ```suggestion
* *Ubuntu 16.04 Xenial only.* Python 3.6 is not available in the base Ubuntu Xenial distribution. Python 3.6 must be available before you upgrade |st2| packages, but you can add the unofficial 3rd party `Python PPA repository <https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa>`_: which contains packages for Python 3.6.
```

    # ensure python3.6 package exists and could be installed
    apt-cache show python3.6

*  *Ubuntu 16 and RHEL/CentOS 7 only*. All packs installed prior to upgrade will need to have their virtual environment re-created after upgrading |st2| packages (on all nodes which run st2actionrunner services), using the following commands:

.. sourcecode:: bash

>     sudo rm -r /opt/stackstorm/virtualenvs

Reviewer 2: Do we also need ``-f`` flag?

Reviewer 2: Also, I guess this will also delete whole ``virtualenvs`` directory, not sure if that's a problem or not.

Perhaps ``rm -rf /opt/stackstorm/virtualenvs/*`` is better.

Author: It worked on my environment without the -f, but I've updated to the version you mentioned, as agree it's better, and no harm in using the -f in case there are files in some environments that need it.
